src/gui/qgsexpressionbuilderwidget.cpp

```diff
@@ -628,9 +628,9 @@ void QgsExpressionBuilderWidget::txtExpressionString_textChanged()
   if ( text.isEmpty() )
   {
     lblPreview->clear();
-    lblPreview->setStyleSheet( QLatin1String( "" ) );
```

Comment:
We really should add `QLatin1String( "" )` and `QStringLiteral( "" )` to the banned keywords test